Chakra failed with error C2440 when build with permissive- by msvc on Windows

Chakra failed with error C2440 when build with permissive- by msvc on Windows, I use latest version a3737b0 on master branch. Could you please help take a look at this? Noted that this issue only found when compiles with unreleased vctoolset, that next release of MSVC will have this behavior.

**You can repro this issue as the steps below:**

1. git clone -c core.autocrlf=true https://github.com/microsoft/ChakraCore D:\Chakra\src
2. open a clean x86 prompt (C:\windows\syswow64\cmd.exe) and browse to D:\Chakra\src
3. set _CL_=/d1ParseTrees:identErrors- /permissive-
4. msbuild /m /p:Platform=x86 /p:Configuration=Test /p:WindowsTargetPlatformVersion=10.0.17134.0 Build\Chakra.Core.sln /t:Rebuild

**Error info:**
d:\chakra\src\bin\nativetests\biguinttest.cpp(235): error C2440: '=': cannot convert from 'const char [11]' to 'char *'
d:\chakra\src\bin\nativetests\biguinttest.cpp(235): note: Conversion from string literal loses const qualifier (see /Zc:strictStrings)
d:\chakra\src\bin\nativetests\biguinttest.cpp(247): error C2440: '=': cannot convert from 'const char [11]' to 'char *'
d:\chakra\src\bin\nativetests\biguinttest.cpp(247): note: Conversion from string literal loses const qualifier (see /Zc:strictStrings)
d:\chakra\src\bin\nativetests\biguinttest.cpp(261): error C2440: '=': cannot convert from 'const char [21]' to 'char *' d:\chakra\src\bin\nativetests\biguinttest.cpp(261): note: Conversion from string literal loses const qualifier (see /Zc:strictStrings)

--- a/bin/NativeTests/BigUIntTest.cpp
+++ b/bin/NativeTests/BigUIntTest.cpp
@@ -226,7 +226,7 @@ namespace BigUIntTest
     TEST_CASE("Init_From_Char_Of_Digits", "[BigUIntTest]")
     {
         BigUInt biDec;
-        char *charDigit;
+        const char *charDigit;
         bool result;
         int charDigitLength;
 

